starport/services/network/init.go

```diff
@@ -88,6 +88,16 @@ func (b *Blockchain) InitAccount(ctx context.Context, v chain.Validator, account
 
 // initGenesis creates the initial genesis of the genesis depending on the initial genesis type (default, url, ...)
 func (b *Blockchain) initGenesis(ctx context.Context) error {
+	genesisPath, err := b.chain.GenesisPath()
+	if err != nil {
+		return err
+	}
+
+	// remove existing genesis
+	if err := os.RemoveAll(genesisPath); err != nil {
+		return err
+	}
+
 	// if the blockchain has a genesis URL, the initial genesis is fetched from the url
 	// otherwise, default genesis is used, which requires no action since the default genesis is generated from the init command
```

Comment:
```suggestion
	// if the blockchain has a genesis URL, the initial genesis is fetched from the URL
	// otherwise, the default genesis is used, which requires no action since the default genesis is generated from the init command
```